src/utils/registration-helpers.ts

```diff
@@ -704,6 +704,17 @@ export const openFileInNewTab = (fileUri: string) => {
   }
 };
 
+const hasChangedContributors = (persistedContributors: Contributor[], updatedContributors: Contributor[]): boolean => {
+  if (persistedContributors.length !== updatedContributors.length) {
+    return true;
+  }
+
+  const sortedPersistedContributors = persistedContributors.map((contributor) => contributor.identity.id).sort();
+  const sortedUpdatedContributors = updatedContributors.map((contributor) => contributor.identity.id).sort();
+
+  return !sortedPersistedContributors.every((id, index) => id === sortedUpdatedContributors[index]);
```

Comment:
Her trenger vi ikke å sjekke alle, når vi finner ett avvik kan vi konkludere
```suggestion
  return sortedPersistedContributors.some((id, index) => id !== sortedUpdatedContributors[index]);
```